packages/sonarwhal/tests/lib/cli/rules/fixtures/new.txt

```diff
@@ -5,7 +5,7 @@
 import { Category } from '../../enums/category';
 import { RuleContext } from '../../rule-context';
 // The list of types depends on the events you want to capture.
-import { IRule, IRuleBuilder, IFetchStart, IFetchEnd, IFetchError } from '../../types';
+import { IRule, IRuleBuilder, FetchStart, FetchEnd, FetchError } from '../../types';
```

Comment:
Are we using this file? because it is an old version of a rule, but tests are not failing...

Reply:
@molant ⬆️